docs/_config.yml

```diff
@@ -24,6 +24,18 @@ defaults:
     values:
       layout: "default"
 
+t:
```

Comment:
i can see why keeping this identifier short is useful (its used as a lookup hash in various templates). how about leaving a comment above this line explaining how this is used?